### PR TITLE
[Concept Entry] ransomware

### LINKED ISSUE #2010: [Concept Entry] Cybersecurity Ransomware
### Reason/inspiration (optional)

We would like a new term entry for [Cybersecurity](https://www.codecademy.com/resources/docs/cybersecurity) about Ransomware 

The file should be located at `docs/content/cybersecurity/concepts/ransomware/ransomware.md`

The entry should: 

- Describe the concept of ransomware. 
- Explain baisc countermeasures to fight ransomware. 
- Link mentions of other cybersecurity concepts to their entries where appropriate. 

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entrty for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by JohnSteeleCG**
### Description

Created concept entry for Ransomware located at docs/content/cybersecurity/concepts/ransomware/ransomware.md
added brief description on ransomware and various countermeasures. Closes #2010 if merged.

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/cybersecurity/concepts/ransomware/ransomware.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+- Implement the use of [Multi-factor Authentication](https://www.codecademy.com/resources/docs/cybersecurity/multi-factor-authentication) to ensure that an attacker will not gain access to important accounts online by verifying an additional factor, such as a token, that the authentic user would only possess.
+- Avoid downloading software from suspicious websites, as they could contain ransomware hidden in their files. It is recommended to only download software and updates from trusted official support websites.
+- Avoid opening suspicious links or documents in emails sent by an unknown sender. Attackers will hide ransomware in emails in the hopes that someone opens them and infects the device that they are on.
+- Use up-to-date security software to protect devices. Outdated software might contain [Vulnerabilities](https://www.codecademy.com/resources/docs/cybersecurity/vulnerability) making it easy for the attacker to infect the device.
```
Comment (SSwiniarski) [inline content/cybersecurity/concepts/ransomware/ransomware.md:25]:
```suggestion
- Use up-to-date security software to protect devices. Outdated software might contain [vulnerabilities](https://www.codecademy.com/resources/docs/cybersecurity/vulnerability) making it easy for the attacker to infect the device.
```

File: content/cybersecurity/concepts/ransomware/ransomware.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+## Preventing Ransomware
+
+There are various basic countermeasures to avoid ransomware from infecting a device:
+
+- Use secured networks when possible. Using unsecured networks will leave the device unprotected and increase the [risk](https://www.codecademy.com/resources/docs/cybersecurity/risk) of ransomware infecting the device. The use of a [Virtual Private Network](https://www.codecademy.com/resources/docs/cybersecurity/virtual-private-network) is recommended to secure and encrypt the connection to protect against malicious threats.
+- Perform frequent backups of important data and store them on a separate device that cannot be accessed from the network. If a device is infected with ransomware, it is possible to restore the device to a previous state before the infection occurred.
+- Implement the use of [Multi-factor Authentication](https://www.codecademy.com/resources/docs/cybersecurity/multi-factor-authentication) to ensure that an attacker will not gain access to important accounts online by verifying an additional factor, such as a token, that the authentic user would only possess.
```
Comment (SSwiniarski) [inline content/cybersecurity/concepts/ransomware/ransomware.md:22]:
```suggestion
- Implement the use of [multi-factor authentication](https://www.codecademy.com/resources/docs/cybersecurity/multi-factor-authentication) to ensure that an attacker will not gain access to important accounts online by verifying an additional factor, such as a token, that the authentic user would only possess.
```

File: content/cybersecurity/concepts/ransomware/ransomware.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  - 'paths/fundamentals-of-cybersecurity'
+---
+
+**Ransomware** is a [cyber attack](https://www.codecademy.com/resources/docs/cybersecurity/cyber-attack) that blocks and prevents a user from accessing important files or personal data that is located on a device or computer.
+
+## Preventing Ransomware
+
+There are various basic countermeasures to avoid ransomware from infecting a device:
+
+- Use secured networks when possible. Using unsecured networks will leave the device unprotected and increase the [risk](https://www.codecademy.com/resources/docs/cybersecurity/risk) of ransomware infecting the device. The use of a [Virtual Private Network](https://www.codecademy.com/resources/docs/cybersecurity/virtual-private-network) is recommended to secure and encrypt the connection to protect against malicious threats.
```
Comment (SSwiniarski) [inline content/cybersecurity/concepts/ransomware/ransomware.md:20]:
```suggestion
- Use secured networks when possible. Using unsecured networks will leave the device unprotected and increase the [risk](https://www.codecademy.com/resources/docs/cybersecurity/risk) of ransomware infecting the device. The use of a [virtual private network (VPN)](https://www.codecademy.com/resources/docs/cybersecurity/virtual-private-network) is recommended to secure and encrypt the connection to protect against malicious threats.
```

File: content/cybersecurity/concepts/ransomware/ransomware.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  - 'Cybersecurity'
+  - 'Computer Science'
+Tags:
+  - 'Cybersecurity'
+CatalogContent:
+  - 'introduction-to-cybersecurity'
+  - 'paths/fundamentals-of-cybersecurity'
+---
+
+**Ransomware** is a [cyber attack](https://www.codecademy.com/resources/docs/cybersecurity/cyber-attack) that blocks and prevents a user from accessing important files or personal data that is located on a device or computer.
```
Comment (SSwiniarski) [inline content/cybersecurity/concepts/ransomware/ransomware.md:14]: Add the fact that the bad actors will ask for money, usually cryptocurrency, in order to restore the compromised system.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@JohnSteeleCG, Looks good. I left a few comments. Other than the line-edit things, I want you to add a sentence or two about the "ransom" part of ransomware.

**Comment by JohnSteeleCG** [comment]
@SSwiniarski made the requested changes, let me know if it looks good.

**Review by SSwiniarski — APPROVED** [review]
@JohnSteeleCG Looks good for another review.

**Comment by SSwiniarski** [comment]
@JohnSteeleCG congrats on the new entry. Live link is here:https://www.codecademy.com/resources/docs/cybersecurity/ransomware

image

**Pull request merged**